### PR TITLE
Adding a glossary which provides the user with information on the metrics

### PR DESCRIPTION
**Pull request by MichaelJamesStam**

### CODE REVIEW

File: front-end/src/Pages/Glossary.elm [hunk 1]
```diff
@@ -0,0 +1,1162 @@
...
+            [ column [ spacing 20 ]
+                [ paragraph [] [ text """DE-STRESS uses a wide range of metrics that come from 
+                                         software developed by researchers all across
+                                         the world and are well used in protein design. However, due to the 
+                                         large numbers of metrics from different sources, it can be difficult 
+                                         to keep track of what these different metrics mean and 
+                                         how to use them. This page has been created to help you understand 
+                                         what these metrics mean, where they have came from and their convention 
+                                         for use.""" ]
+                , paragraph [] [ text """The first table gives a description of the different software
```
Comment (ChrisWellsWood) [inline front-end/src/Pages/Glossary.elm:1060]: "programmes" instead of software in this section might help the text to flow a bit better.

**Review by ChrisWellsWood — CHANGES_REQUESTED** [review]
This is looking great @MichaelJamesStam, it's running fine on my machine. Just one small point on the text (below) and a query: Is there not version information for the other programmes (DSSP, DFIRE2 etc)?

File: front-end/src/Pages/Glossary.elm (hunk 1 above)
Comment (MichaelJamesStam) [reply front-end/src/Pages/Glossary.elm]: I think I managed to find the DSSP version number just from the file name. EvoEF2 just has the version EvoEF V2 and I can't find anything for dfire2 (dfire-pair). I've checked the readme and looked in the code. I've also updated the text a bit as I used the word "different" quite a lot and changed software to programme.

**Review by ChrisWellsWood — APPROVED** [review]
Looking great, merging.

**Pull request merged**